src/index.tsx

```diff
@@ -1,21 +1,29 @@
-import * as React from "react";
-import { useState, useEffect, useMemo, useRef } from "react";
+import React, { useCallback, useReducer, useRef } from "react";
+import { useState, useEffect } from "react";
```

Comment:
I'd love some automated JS tool to organize imports. 

But outside of that, it would be nice to try to keep the style of the existing imports. Can we either combine these imports into one or keep the star-import and named import separate?

Reply:
Apologies I missed this, I reverted to the previous styles.